CMakeLists.txt

```diff
@@ -131,6 +131,7 @@ ExternalProject_Add(
         -DNGRAPH_TOOLS_ENABLE=FALSE 
         -DNGRAPH_DEX_ONLY=TRUE         
         -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+        -DNGRAPH_GPU_ENABLE=TRUE
```

Comment:
Can we change this to `-DNGRAPH_GPU_ENABLE=${NGRAPH_GPU_ENABLE}`? Unfortunately not all of our systems are set up to build the GPU backend.

Reply:
fixed.